django/contrib/redirects/migrations/0002_auto_20200508_1537.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1.dev20200507174748 on 2020-05-08 10:07
```

Comment:
This migration should not be called 'auto', plus we drop the "Generated by" comments in Django core.